### PR TITLE
Safari browser compatibility

### PR DESCRIPTION
**Pull request by k88hudson**
This should fix the safari browser compatibility issues, I haven't figured out how to properly get the shim workign with webpack yet though

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
   "dependencies": {
     "classnames": "1.2.1",
     "color": "0.8.0",
+    "intl": "^1.0.0",
```
Comment (gvn) [inline package.json]: Should we lock this down like the other deps?

File: src/pages/project/project.jsx [hunk 2]
```diff
@@ -5,6 +5,7 @@
 //   Try to use external shared mixins/components instead.
 // *****************************************************************************
 //
+//
```
Comment (gvn) [inline src/pages/project/project.jsx]: Heh...was this intentional?

File: src/html/index.html [hunk 3]
```diff
@@ -8,6 +8,7 @@
 </head>
 <body>
   <div id="app"></div>
+  <script src="../../js/Intl.js"></script>
```
Comment (gvn) [inline src/html/index.html:11]: Be sure to file a follow up ticket if you haven't already. :)

**Comment by gvn** [comment]
A couple q's, but this is looking good!

File: src/pages/project/project.jsx (hunk 2 above)
Comment (k88hudson) [inline src/pages/project/project.jsx]: oops!

File: package.json (hunk 1 above)
Comment (k88hudson) [inline package.json]: yeah that's a good idea

File: src/html/index.html (hunk 3 above)
Comment (k88hudson) [inline src/html/index.html:11]: https://github.com/mozilla/webmaker-core/issues/363

**Comment by gvn** [comment]
R+

**Pull request merged**